frontend/app/(tabs)/index.tsx

```diff
@@ -60,6 +62,7 @@ export default function HomeScreen() {
           <ThemedText type="defaultSemiBold">app</ThemedText> to{" "}
           <ThemedText type="defaultSemiBold">app-example</ThemedText>.
         </ThemedText>
+        <OnboardButton onPress={() => router.push("/onboard")} text="Onboard" />
```

Comment:
Can you link this button to the login screen where it says sign up? Just to future proof it so that when we get rid of the original screen the screens are already linked up.